openmdao/core/system.py

```diff
@@ -3299,9 +3318,9 @@ def get_io_metadata(self, iotypes=('input', 'output'), metadata_keys=None,
                     ret_meta = None
                 else:
                     if metadata_keys is None:
-                        ret_meta = meta.copy()
+                        ret_meta = _MetadataDict(meta.copy())
```

Comment:
I don't think you need the .copy here anymore.

Reply:
hmm yeah, I guess you're right.. the copy constructor and the `copy()` function both do a shallow copy